pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -3486,6 +3486,28 @@ self: {
        broken = true;
      }) {};
 
+  "ConClusion" = callPackage
```

Comment:
Could you rebase this branch on the `haskell-updates` branch?  We try to have all haskell-related changes go to that branch.

Also, if you rebase on `haskell-updates`, I'm guessing you can drop all the changes from `hackage-packages.nix`, since they should already be available on that branch?

Reply:
Done.